root:2023-12-20 21:01:57,002:ERROR:Mauvais mot de passe!
root:2023-12-20 21:02:09,651:ERROR:Mauvais mot de passe!
root:2023-12-20 21:02:21,078:ERROR:Mauvais mot de passe!
root:2023-12-20 21:02:35,473:ERROR:Mauvais mot de passe!
root:2023-12-21 00:18:02,868:ERROR:Mauvais mot de passe!
root:2023-12-21 00:26:39,496:ERROR:bloc introuvable. peut-étre un fichier corrompu
root:2023-12-21 00:30:06,722:ERROR:bloc introuvable. peut-étre un fichier corrompu
root:2023-12-21 00:40:16,223:ERROR:mot de passe incorrect!
root:2023-12-21 00:40:23,165:ERROR:bloc introuvable. peut-étre un fichier corrompu
root:2023-12-22 10:32:15,286:ERROR:[Errno 2] No such file or directory: '/home/Desktop/assia1/fich2'
root:2023-12-22 10:38:00,363:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-22 10:38:42,492:ERROR:Mauvais mot de passe
root:2023-12-22 10:38:49,943:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-22 10:41:26,840:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-22 10:43:37,578:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-22 10:45:26,778:ERROR:Fichier introuvable!
root:2023-12-22 10:45:42,062:ERROR:Mauvais mot de passe
root:2023-12-22 10:45:52,459:ERROR:Mauvais mot de passe
root:2023-12-22 10:46:02,836:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-22 10:58:58,599:ERROR:Mauvais mot de passe
root:2023-12-22 10:59:09,402:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-22 11:07:52,407:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-22 11:17:53,411:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-22 11:18:36,445:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-22 11:19:47,375:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-22 20:15:03,415:ERROR:mot de passe incorrect!
root:2023-12-22 20:21:50,500:ERROR:mot de passe incorrect!
root:2023-12-22 20:25:10,567:ERROR:mot de passe incorrect!
root:2023-12-23 00:09:20,712:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-23 00:13:02,684:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-23 00:13:49,137:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-23 00:16:38,530:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-23 00:18:15,825:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-23 00:20:15,960:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-23 00:24:42,741:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-23 00:38:47,153:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-23 00:45:07,119:ERROR:Erreur lors de la suppression du fichier assia: 'serveur_metreService' object has no attribute 'suprimerr'

========= Remote Traceback (1) =========
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/rpyc/core/protocol.py", line 359, in _dispatch_request
    res = self._HANDLERS[handler](self, *args)
  File "/usr/local/lib/python3.10/dist-packages/rpyc/core/protocol.py", line 853, in _handle_getattr
    return self._access_attr(obj, name, (), "_rpyc_getattr", "allow_getattr", getattr)
  File "/usr/local/lib/python3.10/dist-packages/rpyc/core/protocol.py", line 781, in _access_attr
    return accessor(obj, name, *args)
AttributeError: 'serveur_metreService' object has no attribute 'suprimerr'

root:2023-12-23 00:46:23,962:ERROR:Une erreur s'est produite, fichier introuvable!
root:2023-12-23 00:47:56,190:ERROR:Mauvais mot de passe
root:2023-12-23 00:48:38,278:ERROR:Fichier introuvable!
root:2023-12-23 00:49:56,277:ERROR:Fichier introuvable!
root:2023-12-23 00:50:15,338:ERROR:Fichier introuvable!
root:2023-12-23 00:50:34,229:ERROR:Fichier introuvable!
root:2023-12-23 00:54:14,690:ERROR:Fichier introuvable!
